### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by snakeye**

### CODE REVIEW

**Comment by JackuB** [comment]
Neat! Thanks a lot! This is a continuation of #168 right?

I'd ask for a single change here before merging: this project is using a [semantic release](https://github.com/semantic-release/semantic-release). It requires a specific commit message format so it can take care of versioning and releasing packages ([see release history](https://github.com/JackuB/apish/releases)).

Could you please update the commit messages (maybe squashing them) to make them look like this: https://www.conventionalcommits.org/ also remove the version from package.json, as that will get generated by semantic release as well.
If you need any help with it, just let me know, I can take care of it.

File: src/mock.js [hunk 1]
```diff
@@ -77,14 +77,30 @@ const mock = (refract, options) => {
                   responseBody,
                   responseHeaders
                 };
-                mocks.push(mockRoute(nockOptions));
+
+              if(!routes.hasOwnProperty(resourceUrl)) {
```
Comment (JackuB) [inline src/mock.js]: This bit looks misaligned

File: src/mock.js [hunk 2]
```diff
@@ -77,14 +77,30 @@ const mock = (refract, options) => {
...
 
+    // sort URLs by length
+    const urls = Object.keys(routes);
+    urls.sort((a, b) => b.length - a.length);
+
+    // collect mocks
+    const mocks = [];
+    urls.forEach(url => {
+      const resources = routes[url];
+      resources.forEach(resource => {mocks.push(mockRoute(resource))})
```
Comment (JackuB) [inline src/mock.js]: You could drop curly braces here

```suggestion
      resources.forEach(resource => mocks.push(mockRoute(resource)))
```

File: src/mock.js (hunk 1 above)
Comment (snakeye) [reply src/mock.js]: Fixed.

File: src/mock.js (hunk 2 above)
Comment (snakeye) [reply src/mock.js]: Fixed.

**Comment by snakeye** [comment]
Yes, it's continuation of #168 but I made it as separate PR to be merged afterwards as it is not so critical.

I have fixed the issues and added correct commit messages.

**Pull request merged**